tx/textual/valuerenderer/enum.go

```diff
@@ -0,0 +1,27 @@
+package valuerenderer
+
+import (
+	"context"
+	"io"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+type enumRenderer struct{}
+
+var _ ValueRenderer = (*enumRenderer)(nil)
+
+func (er enumRenderer) Format(_ context.Context, v protoreflect.Value, w io.Writer) error {
+	// TODO: Decide if we should be serializing the type and full package
+	// path of the enum value so as to unmarshal it its .Parse method.
+	formatted, err := formatInteger(v.String())
```

Comment:
I believe the adr specifies a high-level algorithm, which basically would transform the `VoteOption` enum values from `VOTE_OPTION_YES` to `Yes`.

I think we can keep this algorithm, though it would probably be a good idea to update the ADR in this PR with a some better specs.